FrontEnd/Hopsy/src/screens/ProfileScreen.js

```diff
@@ -29,19 +29,19 @@ export default class ProfileScreen extends React.Component {
           username: GLOBAL.user,
         })
     })  
-    .then((response) => response.text())
-      .then((responseJson) => {
-        this.setState({
-          fullname: responseJson,
-        }, function() {
-        })
-      })
+    .then(response => {
+      this.setState({
+        fullname: JSON.parse(response),
+      }, function() {
```

Comment:
why is there an empty fuction after the setstate

Reply:
Blank function removed in https://github.com/nauticalmonkey/CSC308/commit/9273d5ca9b47ebdbb5da410f31ace01fe27bcb11